src/api-service/__app__/onefuzzlib/azure/containers.py

```diff
@@ -19,47 +28,37 @@
     choose_account,
     get_accounts,
     get_storage_account_name_key,
+    get_storage_account_name_key_by_name,
 )
 
 
 @cached
-def get_blob_service(account_id: str) -> BlockBlobService:
+def get_blob_service(account_id: str) -> BlobServiceClient:
     logging.debug("getting blob container (account_id: %s)", account_id)
     account_name, account_key = get_storage_account_name_key(account_id)
-    service = BlockBlobService(account_name=account_name, account_key=account_key)
-    return service
-
-
-def get_service_by_container(
-    container: Container, storage_type: StorageType
-) -> Optional[BlockBlobService]:
-    account = get_account_by_container(container, storage_type)
-    if account is None:
-        return None
-    service = get_blob_service(account)
+    account_url = f"https://{account_name}.blob.core.windows.net/"
```

Comment:
Is the `blob.core.windows.net` part exposed as a default via the SDK, in any way?

Reply:
I looked and didn't see a viable method.  Note, this follows the SDK's examples & tests:

https://github.com/Azure/azure-sdk-for-python/blob/f8be630ec739324fbc540083f4aa3c4e71869f3f/sdk/storage/azure-storage-blob/samples/blob_samples_authentication.py#L31-L33

https://github.com/Azure/azure-sdk-for-python/blob/f8be630ec739324fbc540083f4aa3c4e71869f3f/sdk/storage/azure-storage-blob/tests/test_container.py#L1044-L1046